docs/content/start/property-dds/property-overview.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Overview
+menuPosition: 1
+---
+
+Property DDS represents the managed data in a typed, hierarchical data model called a *PropertySet*. This model has many
+similarities to JSON, but is a richer model, which adds more fine-grained types, additional collection types,
+references and gives the ability to use schemas to describe the structure of properties.
+
+## Data Model
+
+A PropertySet is a tree structured data model in which every node of the tree is a property that can be of one of the
+following categories:
+
+**Primitive Property**
+  A primitive property is a leaf in the property tree that stores data and does not have any children of its own.
+  It has a type that defines which data is stored in the leaf (e.g. ``String``, ``UInt32``, ``Boolean``).
+
+**Collection**
+  Property DDS supports three basic types of collections: ``map``, ``set`` and ``array``. These collections can either contain
+  other property objects or they can directly contain primitive values.
+
+**Container Property**
+  A container property is a property that has child properties. It can either have a fixed structure, which is defined
+  via a [Schema]({{< ref "property-schemas.md" >}}), or it can be a *NodeProperty*, which has a dynamic structure. It is also possible to
+  dynamically add additional properties to a property with a schema that inherits from *NodeProperty*.
+
+**Reserved Property**
+  A reserved property is a container property whose structure is predefined by Property DDS.
+
+![Example of a PropertySet tree](/images/property_example_data_model.png)
+
+
+## ProperySet Root
+
+The root of the *PropertySet* hierarchy can be accessed through the root property of the *Property DDS*
+
+```javascript
+var rootProperty = myPropertyDDS.root;
+```
+
+The root property is of type `NodeProperty` from the root you can traverse the hierarchy and make changes to values,
+insert and remove properties.
+
+## Add Data to the PropertySet
+
+```javascript
+var stringProperty = PropertyFactory.create('String');
+
+// Set it’s value to 'Hello World!'
+stringProperty.setValue('Hello World!');
+
+
+// Add it to the root property under the id 'myProperty'
+rootProperty.insert('myProperty', stringProperty);
+
+console.log('Added one property to the root property');
+rootProperty.prettyPrint();
+```
+
+### What is Property Factory?
+
+The is a factory object for instantiating Properties. It comes with a set of rich primitive types (Float32, Enum, Uint16
+, Uint32, etc. and custom types) For a more detailed explanation please refer to the
```

Comment:
```suggestion
, Uint32, etc. and custom types). For a more detailed explanation please refer to the
```

Reply:
@kailui - Thank you for contributing proof reading fixes!